xep-0425.xml

```diff
@@ -182,8 +194,8 @@
 
 <xs:schema
     xmlns:xs='http://www.w3.org/2001/XMLSchema'
-    targetNamespace='urn:xmpp:message-moderate:0'
-    xmlns='urn:xmpp:message-moderate:0'
+    targetNamespace='urn:xmpp:message-moderate:1'
+    xmlns='urn:xmpp:message-moderate:1'
     elementFormDefault='qualified'>
 
   <xs:element name='moderate'></xs:element>
```

Comment:
Nit: Shouldn't the schema gain an `id` attribute now?

Reply:
Thanks, added.